src/uu/numfmt/numfmt.md

```diff
@@ -1,4 +1,3 @@
-<!-- spell-checker:ignore N'th M'th -->
```

Comment:
Why did you delete this? Now the spell checker is complaining :)

Reply:
I deleted it in the wrong file, so sorry about that

Reply:
You haven't undone the change yet :)